packages/db/index.js

```diff
@@ -438,7 +438,7 @@ export class DBClient {
   /**
    * Upsert given pin status.
    *
-   * @param {Array<import('./db-client-types').PinUpsertInput>} pins
+   * @param {Array<import('./db-client-types').PinsUpsertInput>} pins
    */
   async upsertPins (pins) {
     const { error } = await this._client
```

Comment:
We need to make sure we're mapping the objet to the db columns names here if we stick with using the client.

Reply:
Object to column mapping added.